pkg/actions/cluster/owned.go

```diff
@@ -105,8 +110,14 @@ func (c *OwnedCluster) Delete(_ time.Duration, wait, force bool) error {
 		if err != nil {
 			return err
 		}
-		if err := drainAllNodegroups(c.cfg, c.ctl, c.stackManager, clientSet, allStacks); err != nil {
-			return err
+
+		nodeGroupManager := c.newNodeGroupManager(c.cfg, c.ctl, clientSet)
+		if err := drainAllNodeGroups(c.cfg, c.ctl, clientSet, allStacks, disableNodegroupEviction, nodeGroupManager, attemptVpcCniDeletion); err != nil {
+			if force {
+				logger.Warning("error occurred during nodegroups draining")
```

Comment:
```suggestion
				logger.Warning("error %q occurred during nodegroups draining, force=true so proceeding with deletion", err.Error())
```

probably find better wording :laughing: but the improvement makes sense :smile:

Reply:
Done